payments-core/res/values/strings.xml

```diff
@@ -185,15 +185,6 @@
     <string name="stripe_paymentsheet_or_pay_using" tools:ignore="MissingTranslation">Or pay using</string>
     <string name="stripe_paymentsheet_expiration_date_hint">MM / YY</string>
 
-    <!-- Title for card payment method -->
```

Comment:
are these not translated? looks like we do have other strings like`stripe_paymentsheet_add_payment_method_card_information` translated

Reply:
They will be translated, we just wait to batch translate before a major release.